src/features/mangaViewer/utils/utility.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file going to be used in the future? Just wondering about the empty file.

Reply:
Thank you for that, probably just forgot to delete it.